examples/neural_compressor/question-answering/run_qa.py

```diff
@@ -802,15 +806,18 @@ def train_func(model):
             q8_config.set_config("quantization.approach", quant_approach)
 
         quant_approach = IncQuantizationMode(q8_config.get_config("quantization.approach"))
+        calib_dataloader = trainer.get_train_dataloader() if quant_approach == IncQuantizationMode.STATIC else None
         # torch FX used for post-training quantization and quantization aware training
         # dynamic quantization will be added when torch FX is more mature
         if quant_approach != IncQuantizationMode.DYNAMIC:
             if not training_args.do_train:
                 raise ValueError("do_train must be set to True for static and aware training quantization.")
+            if optim_args.backend == "ipex":
+                q8_config.set_config("model.framework", "pytorch_ipex")
+                calib_dataloader = trainer.get_eval_dataloader()
```

Comment:
Why is the calibration dataset taken from the evaluation dataset when IPEX is enabled ?